repos/system_upgrade/common/actors/scankernelcmdline/tests/test_scankernelcmdline.py

```diff
@@ -0,0 +1,37 @@
+from leapp.libraries.actor import scankernelcmdline
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api
+from leapp.models import KernelCmdline, KernelCmdlineArg
+
+
+def test_cmdline_output(monkeypatch):
+
+    mocked_cmd_output = 'BOOT_IMAGE=(hd0,msdos1)/vmlinuz-xxx \
+root=/dev/mapper/guest \
+ro \
+console=tty0 \
+console=ttyS0,115200 \
+rd_NO_PLYMOUTH \
+biosdevname=0 net.ifnames=0 \
+crashkernel=auto'
+
+    current_actor = CurrentActorMocked(src_ver='8.10', dst_ver='9.6')
+    monkeypatch.setattr(api, 'current_actor', current_actor)
+    monkeypatch.setattr(api, 'produce', produce_mocked())
+    scankernelcmdline.parse_cmdline_input(mocked_cmd_output)
+
+    expected_params = [KernelCmdlineArg(key=k, value=v) for k, v in [
+        ('BOOT_IMAGE', '(hd0,msdos1)/vmlinuz-xxx'),
+        ('root', '/dev/mapper/guest'),
+        ('ro', None),
+        ('console', 'tty0'),
+        ('console', 'ttyS0,115200'),
+        ('rd_NO_PLYMOUTH', None),
+        ('biosdevname', '0'),
+        ('net.ifnames', '0'),
+        ('crashkernel', 'auto')]]
+
+    expected_output_msg = KernelCmdline(parameters=expected_params)
+
+    assert api.produce.model_instances
+    assert expected_output_msg == api.produce.model_instances[0]
```

Comment:
@karolinku  current test is good, but it's not sufficient. This test would not actually reveal the bug that is fixed by this PR. I suggest either to update the test or create another onw that will test the bug specifically. Both ways are ok for me.

Also, current test is not covering obtaining the information about the kernel cmdline from the system - it tests only the parsing of the obtained data. I am ok with that, but I would add at least a comment to the test that this is not tested with a short explanation why.  In case you would like to cover it, I suggest to move that call also to the library to be able to cover it by unit-tests (so it could be mocked). But as I mentioned, the comment in this case would be enough.